Dockerfiles/cpp/.devcontainer/Dockerfile

```diff
@@ -52,11 +52,8 @@ RUN ./install-python.sh && pip3 install -r /container/cpp/requirements.txt
 WORKDIR /container/cpp/build
 WORKDIR /container/cpp
 ARG TARGETARCH
-RUN bash ./install_dependencies.sh -x $TARGETARCH
-
-# Remove build files for final image
-RUN conan remove --builds --src --force '*'
```

Comment:
why did you remove this cleanup? I mean that would reduce the size of the baseimage as far as I understand

Reply:
This kind of "selective" cleanup isn't supported by Conan 2 any longer.

Reply:
hmm :(